fixtures/redshift_schema_edl.sql

```diff
@@ -0,0 +1,241 @@
+CREATE SCHEMA IF NOT EXISTS {redshift_schema_edl};
+
+--
+-- TODO: EDL cutover?
+--
+-- CREATE TABLE IF NOT EXISTS {redshift_schema_edl}.canvas_api_enrollments
+-- (
+--     course_id VARCHAR NOT NULL,
+--     user_id VARCHAR NOT NULL,
+--     term_id VARCHAR(4) NOT NULL,
+--     last_activity_at TIMESTAMP,
+--     feed TEXT NOT NULL
+-- );
```

Comment:
@johncrossman We shouldn't need EDL data structures for canvas_api_enrollments; the EDL doesn't currently cover Canvas data.